websites/P/Puray Animes/presence.ts

```diff
@@ -35,10 +43,13 @@ presence.on("UpdateData", () => {
 			{
 				label: "Assistir episódio",
 				url: location.href
+			},
+			{
+				label: "Ver anime",
+				url: document.querySelector<HTMLAnchorElement>(".mb-4>a").href
 			}
 		];
-		const video = document.querySelector("video");
```

Comment:
Also why

Reply:
Why not?
I just thought it was better that way

Reply:
How is it better? It will only unnecessarily call the function.

Reply:
ok, i fixed it

Reply:
No, this isn't what I meant. You moved this variable outside and I wanted you to move it back in

Reply:
ok, it's done.